### DISCUSSION TITLE
plugin for itk image viewer

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#648] [Q&A] [answered]
Will there be a MONAI Label plugin for itk-vtk-image viewer or ITK-SNAP? Thanks!

**Comment by SachidanandAlle** [comment, marked as answer]
We have reference implementation for 3d slicer and OHIF.. its a open source.. we look for contributions from corresponding communities for the rest... the server apis are pretty simple to integrate with any clients..

**Reply by hw-ju** [reply]
Thanks @SachidanandAlle !
